src/components/Dashboard/Wishlist/Wishlist.js

```diff
@@ -17,6 +19,16 @@ export class Wishlist extends Component {
     removeProd(productId, userId, token);
   };
 
+  handleClickToCart = (product) => {
```

Comment:
For my taste this logic sould be in the action creator, so we do not need to pass `idsProductsInCart` from store and connect `updateProduct` and `addProduct` actions.
Instead we would have only one connected action.